packages/web-components/fast-components/src/data-grid/data-grid.stories.ts

```diff
@@ -387,7 +387,7 @@ function cellTemplateButtonClick(cell: DataGridCell): void {
     ) {
         return;
     }
-    const newRowData: object = { ...cell.rowData };
+    const newRowData: any = { ...cell.rowData };
```

Comment:
I left `any` here since this is a storybook file, but I can definitely update if someone has a cleaner approach 👍

Reply:
I think this, and the cast `as object` above point to `rowData` being incorrectly typed in `DataGridCell`. Something like `Record<string, unknown>` should work here:

```suggestion
    const newRowData: Record<string, unknown> = { ...cell.rowData };
```

@scomea is there a better type for `DataGridCell.rowData` than `object | null`?